### PR TITLE
[OSCD] split Linux and macOS tests for T1518.001; update processes list

### PR DESCRIPTION
**Pull request by yugoslavskiy**
**Details:**

- split macOS and Linux tests for T1518.001
- update processes lists

**Testing:**
Tested on a local machine

**Associated Issues:**
Here are the relevant Sigma rules:

- https://github.com/Neo23x0/sigma/blob/6f3ac02cb363b0fc68ebe3f762b3b6e6385fa568/rules/linux/lnx_security_software_discovery.yml
- https://github.com/Neo23x0/sigma/blob/5a8c7cd3f9291ac010f2f0e94037b0304d5cf458/rules/linux/macos_security_software_discovery.yml

### CODE REVIEW

File: atomics/T1518.001/T1518.001.yaml [hunk 1]
```diff
@@ -34,19 +34,45 @@ atomic_tests:
...
+    name: sh
+- name: Security Software Discovery - ps (Linux)
+  description: |
+    Methods to identify Security Software on an endpoint
+    when sucessfully executed, command shell  is going to display AV/Security software it is running.
+  supported_platforms:
+  - linux
+  executor:
+    command: |
+      ps aux | grep falcond
```
Comment (rc-grey) [inline atomics/T1518.001/T1518.001.yaml:68]: I'm not sure what is more common. But this is certainly more efficient, on both Linux and MacOS. Again if you feel this is more what is intended let me know and ill approve and merge.

`ps aux | egrep 'falcond|nessusd|cbagentd|td-agent|packetbeat|filebeat|auditbeat|osqueryd'`

**Comment by yugoslavskiy** [comment]
Hello @keithmccammon! could you please tell me why this PR was closed?

**Comment by keithmccammon** [comment]
Hi @yugoslavskiy, it appears we may have improperly coordinated the final merge of the oscd branch. Or, we could have merged but retained it. I didn't realize that we still had an outstanding PR. 

I've restored the [branch](https://github.com/redcanaryco/atomic-red-team/tree/oscd) so that we can complete this PR. 

Apologies for the confusion!

**Comment by yugoslavskiy** [comment]
No problems, @keithmccammon (: I was thinking there is something wrong with the PR.
It seems that there is a couple of more PRs closed the same way:

- https://github.com/redcanaryco/atomic-red-team/pull/1240
- https://github.com/redcanaryco/atomic-red-team/pull/1264

Could you please reopen them as well?

Would you mind if we will create final `oscd` -> `master` PR ourselves, when these PRs will be merged to the `oscd` branch?
We would like to summarise the contribution as we did it last year for the [Sigma project](https://github.com/Neo23x0/sigma/pull/554).

Thank you!

File: atomics/T1518.001/T1518.001.yaml (hunk 1 above)
Comment (yugoslavskiy) [reply atomics/T1518.001/T1518.001.yaml:68]: hey @rc-grey! yes, that will work as well, not so much different from a behavioral perspective, but certainly more efficient. will change it right now

**Comment by keithmccammon** [comment]
> Would you mind if we will create final `oscd` -> `master` PR ourselves, when these PRs will be merged to the `oscd` branch?
> We would like to summarise the contribution as we did it last year for the [Sigma project](https://github.com/Neo23x0/sigma/pull/554).

I merged the branch last night, but we're fine holding it open until you perform the final merge and any summarization.

**Comment by yugoslavskiy** [comment]
Hello @rc-grey! Could you please tell me if there anything else I should fix in the test? (:

**Comment by yugoslavskiy** [comment]
Hello @keithmccammon, @mgraeber-rc! 
Could you please tell me if there is anything else left in the PR that I need to modify or it is ready to be merged to the `oscd` branch?
Thanks!

**Comment by mgraeber-rc** [comment]
Thank you!

**Pull request merged**